src/web/index.ts

```diff
@@ -2,5 +2,9 @@ export { Html } from './Html'
 
 export { Loader } from './Loader'
 
+export { BlenderViewportGizmo } from './BlenderViewportGizmo'
```

Comment:
I know I previously talked about them needing to be here in `web` but as you rightly pointed out, we use `document` in other components not in `web` so in light of that, let's put it in `core` and if anyone complains we can sort it then 👍🏼

Reply:
Also `BlenderViewportGizmo` is a mouthful. Can we just call them `GizmoViewport` and `GizmoViewCube` then they'll sit nicely with `GizmoHelper` it sorta unifies them, what do you think?

Reply:
shorter is better!